graph-builder/src/registry.rs

```diff
@@ -163,13 +224,13 @@ fn find_first_release(
     authenticated_client: dkregistry::v2::Client,
     registry_host: String,
     repo: String,
-    tag: String,
-) -> impl Future<Item = Option<Release>, Error = Error> {
-    let tag_for_error = tag.clone();
+    repo_tag: String,
```

Comment:
this seems a leftover from your local changes. let's just keep the current names

Reply:
No, this is to disambiguate between input and output `repo`.